src/routes/services.ts

```diff
@@ -1,12 +1,17 @@
 import {Router} from 'express';
 import wrap from './wrapper';
 import {servicesController} from '../container';
+import validator from '../middleware/validator';
+import schema from '../validators/schema';
 
 let router = Router({mergeParams: true});
 
 router.route('/')
   .get(wrap(async (req, res, next) => servicesController.list(req, res)))
-  .post(wrap(async (req, res, next) => servicesController.create(req, res)));
+  .post(
+      (req, res, next) => validator(req, res, next, schema.service),
+      wrap(async (req, res, next) => servicesController.create(req, res))
```

Comment:
Mb we can find a better solution for this?
It looks like a lot of code in request classes.
I think that we should use decorators for this.
https://www.typescriptlang.org/docs/handbook/decorators.html
So it will be like:
```ts
class Controller {
    @validate('controller::test')
    public test(request: Request, response: Response): void {
        // ...
    }
}

// validators/controllerTestValidator.ts
export default function validate(validator: Validator, request: Request): boolean {
    // ...
    return true;
}

// validators.ts
import controllerTestValidator from 'validators/controllerTestValidator.ts';

const validators = {
    'controller::test': controllerTestValidator,
    // ...
};

export default validators;
```